src/style/animations.scss

```diff
@@ -0,0 +1,43 @@
+@keyframes moveInLeft {
```

Comment:
universal animations for the prompt display to come in between fetchs to hopefully make it look more fluid on transitions